### PR TITLE
add vscode extension recommendations

### PR DESCRIPTION
**Pull request by Leptopoda**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -2,12 +2,16 @@
   "dart.lineLength": 120,
   "yaml.format.singleQuote": true,
   "dart.flutterSdkPath": ".fvm/flutter_sdk",
+  "files.insertFinalNewline": true,
   // Remove .fvm files from search
   "search.exclude": {
     "**/.fvm": true
   },
   // Remove from file watching
```
Comment (provokateurin) [inline .vscode/settings.json:10]: I hate that they allow comments inside json

**Review by provokateurin — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Leptopoda) [reply .vscode/settings.json:10]: They also allow trailing commas :eyes:

**Review by provokateurin — COMMENTED** [review]
There were no changes in your second push?

**Comment by Leptopoda** [comment]
So now it should also contain the removal of the comments.

**Comment by provokateurin** [comment]
I can still see them here

**Comment by Leptopoda** [comment]
Maybe some bad github UI cache. Can you force reload?

Screenshot 2023-10-01 at 12-46-56 add vscode extension recommendations by Leptopoda · Pull Request #896 · nextcloud_neon

**Review by provokateurin — APPROVED** [review]
Ah ffs the comments were marked red because they were invalid json. When you removed them the line was still red because of the deletion :sweat_smile: The side-by-side view doesn't have this problem of course

**Pull request merged**